cli/cmd/turbo/signals.go

```diff
@@ -1,20 +0,0 @@
-package main
-
-import (
-	"os"
-	"os/signal"
-	"syscall"
-)
-
-func watchSignals(onClose func()) <-chan struct{} {
-	// TODO: platform specific signals to watch for?
```

Comment:
I *think* we're okay with this set of three. I tried to see if there were other signals that we might care about for shutdown, but I think this is fine.

https://pkg.go.dev/os/signal#hdr-Windows